virttest/utils_stress.py

```diff
@@ -1,33 +1,58 @@
 import random
 import time
 import threading
+import logging
 
 from avocado.utils import cpu
 
 from virttest import virsh
 from virttest import utils_hotplug
+from virttest import utils_net
 from virttest.utils_test import libvirt
+from virttest.libvirt_xml.devices.disk import Disk
 
 
 class VMStressEvents():
+
     def __init__(self, params, env):
         """
         :param params: test param
         """
         self.host_cpu_list = cpu.cpu_online_list()
-        self.iterations = int(params.get("stress_itrs", 20))
+        self.iterations = int(params.get("stress_itrs", 1))
         self.event_sleep_time = int(params.get("event_sleep_time", 10))
         self.current_vcpu = params.get("smp", 32)
         self.max_vcpu = params.get("virsh_maxcpus", 32)
         self.ignore_status = params.get("ignore_status", "no") == "yes"
         self.vms = env.get_all_vms()
         self.events = params.get("stress_events", "reboot").split(',')
         self.threads = []
+        self.iface_num = params.get("iface_num", '1')
+        self.iface_type = params.get("iface_type", "network")
+        self.iface_model = params.get("iface_model", "virtio")
+        self.iface_source = eval(params.get("iface_source",
+                                            "{'network':'default'}"))
+        self.attach_option = params.get("attach_option", "")
+        self.detach_option = params.get("detach_option", "")
+        self.strs_net_hotplug = params.get("stress_nethotplug", "no") == "yes"
+        self.stress_test = "yes" == params.get("stress_test", "no")
+        self.disk_size = params.get("virt_disk_device_size", "1")
+        self.disk_type = params.get("disk_type", "file")
+        self.disk_device = params.get("disk_device", "disk")
+        self.disk_format = params.get("disk_format", "qcow2")
+        self.target_bus = params.get("target_bus", "virtio")
+        self.virt_disk_device_size = params.get("virt_disk_device_size", "1")
```

Comment:
this is already assigned `self.disk_size`

Reply:
yes... corrected..